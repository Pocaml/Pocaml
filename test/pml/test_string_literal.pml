let test_cases = [
  "pocaml!";
  "\tpocaml";
  "x :: xs";
  "\"quoted\""
]

let _ = list_map print_endline test_cases
