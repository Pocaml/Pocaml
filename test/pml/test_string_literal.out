pocaml!
	pocaml
x :: xs
"quoted"
